tab-daemon/src/service/pty/scrollback.rs

```diff
@@ -98,6 +98,15 @@ impl ScrollbackBuffer {
     }
 
     pub fn push(&mut self, mut chunk: OutputChunk) {
+        // replace ESC [ 6n, Device Status Report
+        // tbis sequence is echoed as keyboard characters,
```

Comment:
type on in `tbis` maybe `this`?

Reply:
Ha, yep.  Haven't had a chance to self-review this yet.